sdcm/mgmt/cli.py

```diff
@@ -6,20 +6,19 @@
 import json
```

Comment:
this file movement is going to break `siren-tests`
https://github.com/scylladb/siren-tests/blob/master/sct_plugin/cluster_cloud.py#L20

Reply:
I take that back, since `sdcm/mgmt/__init__.py` is importing it